tests/functional/test_ingest_pipeline.py

```diff
@@ -52,7 +51,7 @@ def remove_keys(data, keys_to_remove):
 @pytest.mark.skip(reason="Test environment is not running nv-ingest and redis services.")
 def test_ingest_pipeline():
     client = NvIngestClient(
-        message_client_allocator=RedisClient,
+        message_client_allocator="", #RedisClient,
```

Comment:
Do we just delete this test since it is skipped anyway?

Reply:
Yeah, that probably makes sense. I think I was hoping to fix it, but that can be later.